webapp/nuxt.config.js

```diff
@@ -272,6 +272,26 @@ export default {
           },
         },
       })
+      const tagAttributesForTesting = ['data-test', ':data-test', 'v-bind:data-test']
+      ctx.loaders.vue.compilerOptions = {
+        modules: [
+          {
+            preTransformNode(astEl) {
+              if (!ctx.isDev) {
+                const { attrsMap, attrsList } = astEl
+                tagAttributesForTesting.forEach(attribute => {
+                  if (attrsMap[attribute]) {
+                    delete attrsMap[attribute]
+                    const index = attrsList.findIndex(attr => attr.name === attribute)
+                    attrsList.splice(index, 1)
+                  }
+                })
+              }
+              return astEl
```

Comment:
What does `astEl` stand for? Could we use a more declarative name?

Reply:
I'm not sure... this was a copy :spaghetti: and once it was working, I thought about how it could be improved, but didn't spend much time cause I need to build the image each time to make sure it works.... 
I learned about `delete` from @roschaefer actually. We have some cases in the backend already, really cool stuff!!
this is what `astEl` looks like 
```
{                                                                                                                                                               13:35:28
  type: 1,
  tag: 'signup',
  attrsList: [
    {
      name: ':invitation',
      value: 'true'
    }
  ],
  attrsMap: {
    ':invitation': 'true'
  },
  rawAttrsMap: {},
  parent: {
    type: 1,
    tag: 'ds-card',
    attrsList: [
      [Object]
    ],
    attrsMap: {
      class: 'signup'
    },
    rawAttrsMap: {},
    parent: {
      type: 1,
      tag: 'ds-section',
      attrsList: [],
      attrsMap: {},
      rawAttrsMap: {},
      parent: undefined,
      children: [Array]
    },
    children: []
  },
  children: []
}
```
Any ideas of names? `El` I guess would be short for `Element`... not sure about `ast`...

Reply:
Awww, I was able to find out... it stands for `abstractSyntaxTree`... have updated the code to be more verbose... I love it when people ask for changes I could see myself suggesting!! :heart_eyes_cat:

Reply:
Thank you for researching! 😍